tomviz/python/tomviz/itkutils.py

```diff
@@ -206,11 +207,11 @@ def get_python_voxel_type(dataset):
                 itkTypes.UC: int,
                 itkTypes.US: int,
                 itkTypes.UI: int,
-                itkTypes.UL: long,
+                itkTypes.UL: py2to3.long_type,
```

Comment:
Well, fingers crossed :-) One way or another Python 3 will force longs to be represented by `ints`. The good news is that few images I've come across require long voxel types - certainly nothing acquired by an instrument that I'm aware of.